satpy/composites/sar.py

```diff
@@ -31,16 +31,16 @@
 LOG = logging.getLogger(__name__)
 
 
-def overlay(top, bottom):
+def overlay(top, bottom, maxval=None):
     """Blending two layers.
 
     from: https://docs.gimp.org/en/gimp-concepts-layer-modes.html
     """
-    maxval = xu.maximum(top.max(), bottom.max())
+    if maxval is None:
+        maxval = xu.maximum(top.max(), bottom.max())
```

Comment:
Can we get rid of the `xu` usage?

Reply:
Looks like multiple cases in this module at least.

Reply:
Sure